src/services/sticker.service.ts

```diff
@@ -0,0 +1,503 @@
+/* tslint:disable:max-classes-per-file */
+import { Repository } from '../core/repository';
+import { defaults, snakeCase } from 'lodash';
+import { DateTime } from 'luxon';
+
+export class StickerService extends Repository {
```

Comment:
Hi, @Nerixyz, glad to see you! 😄 
It's a great solution to create the sticker builder 👏
But it's not so great to make it a service, because in fact it is not a service. It doesn't need `IgApiClient` instance in order to work, the builder is stand-alone. We could even make it as separate npm module without `instagram-private-api` as dependency :) I dont offer to do so right now, but i'm thinking about make this lib as `lerna` monorepo because codebase is getting big.

What i offer right now - create `src/sticker-builder` folder (or `stickers`, or any you want) and consider it like npm sub-module. 
Follow the practice `1 class per file` and organise all the `abstract class InstaSticker`  descendants in `sticker-builder/stickers` folder. 
Export all the needed classes and let the users to use it directly, withoit `IgApiClient` instance.